pkgs/tools/networking/network-manager/applet/default.nix

```diff
@@ -40,7 +71,7 @@ in stdenv.mkDerivation rec {
     chmod +x meson_post_install.py # patchShebangs requires executable file
     patchShebangs meson_post_install.py
 
-    substituteInPlace src/wireless-security/eap-method.c --subst-var-by NM_APPLET_GSETTINGS ${glib.makeSchemaPath "$lib" name}
+    substituteInPlace src/wireless-security/eap-method.c --subst-var-by NM_APPLET_GSETTINGS ${glib.makeSchemaPath "$lib" "${pname}-${version}"}
```

Comment:
This was moved to libnma. If we lowercase the variable, we can use `substituteAll`.

Reply:
Yeah, not sure what's up here, as the code is for some reason still there in network-manager-applet, which is why I left it in. Though I'm guessing it's most likely dead code, and on the off chance it's not, it looks like all derivations that needed the patch now use libnma.

Reply:
The code itself does not look dead to me but since there is no library any more, we can just rely on `wrapGAppsHook` and drop the patch here.

Reply:
Right, that makes sense.